qcodes/instrument_drivers/Harvard/Decadac.py

```diff
@@ -115,8 +162,8 @@ def _setvoltage(self, voltage, channel):
             try:
                 self.visa_handle.read()
             except UnicodeDecodeError:
-                logging.warning(" Decadac returned nothing and did nothing. " +
-                                "Please re-run the command")
+                log.warning(" Decadac returned nothing and did nothing. " +
```

Comment:
I got this warning but it HAD changed the voltage like I asked it to and returned the voltage I asked it to, only when first initialising the instrument though, not a big deal but just could be a bit misleading